tests/unit/__init__.py

```diff
@@ -130,6 +131,23 @@ def _secret_not_found(secret_scope, _):
     msg = f"Secret Scope {secret_scope} does not exist!"
     raise NotFound(msg)
 
+    # can't remove **kwargs because it receives format=xxx
```

Comment:
This comment seems to have wrong indentation

Reply:
Ah yes.

Reply:
Fixed.